care/facility/api/viewsets/patient.py

```diff
@@ -426,10 +426,21 @@ def list(self, request, *args, **kwargs):
     @action(detail=True, methods=["POST"])
     def transfer(self, request, *args, **kwargs):
         patient = PatientRegistration.objects.get(external_id=kwargs["external_id"])
```

Comment:
Is auth present in DRY permissions, i cant see any explicit authorization here.

Reply:
This method is inside `PatientViewSet`, which does have the authentication and permission classes:
```
    authentication_classes = [
        CustomBasicAuthentication,
        CustomJWTAuthentication,
        MiddlewareAuthentication,
    ]
    permission_classes = (IsAuthenticated, DRYPermissions)
```

I hope I understood your question correctly. 😅

Reply:
The key i was looking for was AuthZ, its there in DRY,i checked